CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ update DATABASECHANGELOG
 * Added custom thread pool to workaround Slick [deadlocks](https://github.com/slick/slick/issues/1274). The thread pool
 size defaults to the Slick configuration value `db.numThreads`, but may be increased up to Slick's
 `db.maxConnections`, via a new property `actionThreadPoolSize`.
+* Added support for [size](https://github.com/broadinstitute/wdl/blob/master/SPEC.md#float-sizefile-string) WDL standard library function.
```

Comment:
Yeah, IIRC we kinda decided that a lot of this (including what a spec version means) needs to be hashed out and then punted from there. I don't think we can make a snap decision on this and at the moment we have much larger fish to fry so I think what @scottfrazer said is the right thing to do for now.